src/main/scala/org/cloudera/spark/streaming/kafka/KafkaWriter.scala

```diff
@@ -75,51 +75,50 @@ object KafkaWriter {
  * }
  *
  * }}}
- * @param dstream - The [[DStream]] to be written to Kafka
+ * @param dStream - The [[DStream]] to be written to Kafka
  *
  */
-class KafkaWriter[T: ClassTag](@transient dstream: DStream[T]) {
+class KafkaWriter[T: ClassTag](@transient dStream: DStream[T]) {
 
   /**
    * To write data from a DStream to Kafka, call this function after creating the DStream. Once
    * the DStream is passed into this function, all data coming from the DStream is written out to
    * Kafka. The properties instance takes the configuration required to connect to the Kafka
    * brokers in the standard Kafka format. The serializerFunc is a function that converts each
-   * element of the RDD to a Kafka [[KeyedMessage]]. This closure should be serializable - so it
+   * element of the RDD to a Kafka [[ProducerRecord]]. This closure should be serializable - so it
    * should use only instances of Serializables.
    * @param producerConfig The configuration that can be used to connect to Kafka
    * @param serializerFunc The function to convert the data from the stream into Kafka
-   *                       [[KeyedMessage]]s.
+   *                       [[ProducerRecord]]s.
    * @tparam K The type of the key
    * @tparam V The type of the value
    *
    */
   def writeToKafka[K, V](producerConfig: Properties,
-    serializerFunc: T => KeyedMessage[K, V]): Unit = {
+    serializerFunc: T => ProducerRecord[K, V]): Unit = {
```

Comment:
Is `ProducerRecord` available in all versions of 0.8.x? I'd like to make sure we can support slightly older versions as well, since not everyone would have upgraded to the latest and greatest.